manifests/ssltools/create_nssdb.pp

```diff
@@ -0,0 +1,35 @@
+# Creates a nssdb
+class certs::ssltools::create_nssdb inherits certs {
```

Comment:
I'd also prefer if this was just a class with a parameter rather than inheriting from certs. Looks like there are just 2 (nss_db_dir and group).